src/styles/components/_asides.scss

```diff
@@ -189,3 +189,12 @@ $KEY_POINT_COLOR: #6001ff;
 .w-aside--gotchas p {
   margin-top: 8px;
 }
+
+// Colors.
+.w-c--success {
```

Comment:
Remove this here & in codelab.

Reply:
Are you sure we cannot have these new classes to add colors?

Without these classes, the table below looks quite dull and not easy to understand at a glance.

**BEFORE**:
![b](https://on.ahmda.ws/ed1d15/c)

**AFTER**:
![b](https://on.ahmda.ws/a1f2a4/c)

Also this one:


**BEFORE**:
![b](https://on.ahmda.ws/2bb543/c)

**AFTER**:
![b](https://on.ahmda.ws/af4382/c)


Let me know what you think and I'll follow your lead here.
Peace! ✌️

Reply:
@robdodson is the authority here on our styles :)

Reply:
Awesome! @robdodson what do you think? I personally think that graphs and comparison tables are inherently counter-intuitive. I always like to decrease the cognitive load for users with colors, where they can instantly understand something being wrong with `red` colored items and right with the `green` color ones. 

But hey, let me know what you think and I'll follow your lead here. 

Looking forward, peace! ✌️

Reply:
I'm inclined to agree with Katie. We're trying to avoid custom styles in general because, when needed, we want to be able to update site styles with confidence that nothing will break.

Reply:
@mfriesenhahn makes sense. I am all for that. I just need a way to make it easier for the user reading the Codelab to be easily able to differentiate between the good and bad results — by way of red and green colors. 

I was unable to find a class that just changes the colors to the red/green colors defined for web.dev. Are there such classes available that I'd have missed?

Or do you think I should just remove the colors from both the SCSS and the markdown file? 👀

Reply:
I don't think we have specific styles for this kind of UI so I think I'd suggest removing them.

Reply:
Sure, removed in 22aed01